Configuration/AlCa/python/autoCond.py

```diff
@@ -10,17 +10,19 @@
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run1
     'run1_mc_pa'        :   '92X_mcRun1_pA_v2',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
-    'run2_design'       :   '92X_mcRun2_design_v2',
+    'run2_design'       :   '92X_mcRun2_design_v3',
     # GlobalTag for MC production with pessimistic alignment and calibrations for Run2
-    'run2_mc_50ns'      :   '92X_mcRun2_startup_v2',
+    'run2_mc_50ns'      :   '92X_mcRun2_startup_v3',
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
-    'run2_mc'           :   '92X_mcRun2_asymptotic_v2',
+    'run2_mc'           :   '92X_mcRun2_asymptotic_v3',
```

Comment:
The reason of failures now is this v3. This should be v4.

Reply:
done